build/ci-build.yml

```diff
@@ -0,0 +1,78 @@
+name: Build Arcus.WebApi CI
+
+trigger:
+  - master
+pr:
+  - master
```

Comment:
This might be because of me, sorry for that!

However, I'm not sure if we should bother though since we don't use release branches, but can't hurt to build for every PR I guess!

What do you think @stijnmoreels 	?

Reply:
We indeed don't use release braches so it would only be when merging non-master branches. I think that a lot of times these branches are your own branches and therefore I would think that you're responsible for a green build.
I also think that if you such a scenario's happen on local branches and not remote ones; where there isn't any PR-mechanism, only your own disciplne.

That being said, I think it would hurt to run the build for every PR, it's just not directly my way of working where you merge local branches on a remote build system.

Reply:
It does not perse have to be a local branch - If you have a "bigger" feature it sometimes happens that somebody forks your branch and contributes to it via a PR towards your branch to have a review process, however this will most-probably not be the case for Arcus.

However, I don't think it hurts to trigger build for every PR as we've all agreed upon so let's maybe just remove the restriction then?